webapp/messages/en.json

```
@@ -213,7 +213,7 @@
       },
       "unstake-token": "Unstake {symbol}",
       "unstaking": "Unstaking...",
-      "you-can-stake-anytime": "You can withdraw any time and keep the points already earned."
+      "you-can-stake-anytime": "You can unstake any time and keep the points already earned."
```

I just noticed: Given we're here, can we also fix the key? 😅 

```suggestion
      "you-can-unstake-anytime": "You can unstake any time and keep the points already earned."
```

---

Sure thing! Done.